README.md

```diff
@@ -11,15 +12,16 @@
 
 
 ## Prerequisites-
+	
 	1. For routing  - get this package "go get github.com/gorilla/mux"
 	2. For using mongodb - get these packages "gopkg.in/mgo.v2/bson" and "gopkg.in/mgo.v2"
 	
 
 ## For Installing MongoDB follow this Link - 
 [https://docs.mongodb.com/manual/installation](https://docs.mongodb.com/manual/installation)
 
-##To Run Test cases-
- 	There are two ways. The easy one is to use the -run flag and provide a pattern matching names of the tests you want to run.
+## To Run Test cases-
+There are two ways. The easy one is to use the -run flag and provide a pattern matching names of the tests you want to run.
  Example:
  ```
```

Comment:
Mention process for setting current working directory

Reply:
fixed.